### PR TITLE
Release Notes for WebView2 117 Release SDK

### PR DESCRIPTION
**Pull request by mikehoffms**
Rendered article section for review:
https://review.learn.microsoft.com/microsoft-edge/webview2/release-notes?branch=pr-en-us-2810#10204528

Before:
https://learn.microsoft.com/en-us/microsoft-edge/webview2/release-notes

AB#46345087

### CODE REVIEW

File: microsoft-edge/webview2/release-notes.md [hunk 1]
```diff
@@ -117,36 +117,45 @@ For full API compatibility, this version of the WebView2 SDK requires WebView2 R
 
 ###### Bug fixes
 
-* Disabled Mouse Gesture feature by default (Runtime-only) ([Issue 3737](https://github.com/MicrosoftEdge/WebView2Feedback/issues/3737))
-* Fixed a bug where mousewheel scrolling was intermittently broken for visual hosting (Runtime-only)
-* Fixed a bug where downloading APK files in WebView2 crashes WV2 browser process. (Runtime-only) ([Issue 3569](https://github.com/MicrosoftEdge/WebView2Feedback/issues/3569))
+* Disabled the Mouse Gesture feature by default.  (Runtime-only)  ([Issue #3737](https://github.com/MicrosoftEdge/WebView2Feedback/issues/3737))
+* Fixed a bug where mouse wheel scrolling was intermittently broken for visual hosting.  (Runtime-only)
+* Fixed a bug where downloading APK files in WebView2 crashes the WV2<!--todo: spell out as "WebView2"?  lowercase it?--> browser process.  (Runtime-only)  ([Issue #3569](https://github.com/MicrosoftEdge/WebView2Feedback/issues/3569))
```
Comment (mikehoffms) [inline microsoft-edge/webview2/release-notes.md:122]: @victorhuangwq @peiche-jessica - change "the WV2" to "the WebView2 control"?

Comment (victorhuangwq) [reply microsoft-edge/webview2/release-notes.md:122]: that works

File: microsoft-edge/webview2/release-notes.md [hunk 2]
```diff
@@ -117,36 +117,45 @@ For full API compatibility, this version of the WebView2 SDK requires WebView2 R
...
+<!-- ====================================================================== -->
 ## 1.0.2098-prerelease
 
-Release Date: September 18, 2023  <!-- TBD -->
+Release Date: September 18, 2023<!-- todo: finalize -->
 
 [NuGet package for WebView2 SDK 1.0.2098-prerelease](https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.2098-prerelease)
 
-For full API compatibility, this version of the WebView2 SDK requires WebView2 Runtime version 119.0.2098.0 or higher.
+For full API compatibility, this version of the WebView2 SDK requires Microsoft Edge version 119.0.2098.0 or higher.
```
Comment (victorhuangwq) [inline microsoft-edge/webview2/release-notes.md:134]: okay!

File: microsoft-edge/webview2/release-notes.md [hunk 3]
```diff
@@ -104,6 +104,78 @@ For full API compatibility, this version of the WebView2 SDK requires Microsoft
...
+<!-- ====================================================================== -->
+## 1.0.2045.28
+
+Release Date: September 18, 2023<!-- todo: finalize -->
+
+[NuGet package for WebView2 SDK 1.0.2045.28](https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.2045.28)
+
+For full API compatibility, this version of the WebView2 SDK requires WebView2 Runtime version 117.0.2045.28 or higher.
+
+#### General
```
Comment (mikehoffms) [inline microsoft-edge/webview2/release-notes.md:116]:
```suggestion
#### General

###### Promotions

No additional APIs have been promoted to Stable and added in this Release SDK.

```
add this heading and statement like in August Release?  @victorhuangwq @peiche-jessica

Comment (mikehoffms) [inline microsoft-edge/webview2/release-notes.md:116]:
```suggestion
#### All platforms
```
design proposal idea.  When there's a General heading but no .NET or WinRT or Win32/C++ heading at same level like in early releases, the meaning of "General" is lost.  It really means "All platforms", I think.

File: microsoft-edge/webview2/release-notes.md [hunk 4]
```diff
@@ -104,6 +104,78 @@ For full API compatibility, this version of the WebView2 SDK requires Microsoft
...
+## 1.0.2098-prerelease
+
+Release Date: September 18, 2023<!-- todo: finalize -->
+
+[NuGet package for WebView2 SDK 1.0.2098-prerelease](https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.2098-prerelease)
+
+For full API compatibility, this version of the WebView2 SDK requires Microsoft Edge version 119.0.2098.0 or higher.
+
+#### General
+
```
Comment (mikehoffms) [inline microsoft-edge/webview2/release-notes.md:137]:
```suggestion
#### WinRT
```
headings structural design: should this RT-specific info be in a WinRT section at same level as General heading, similar to the headings breakout in early release h2 sections?

Comment (mikehoffms) [reply microsoft-edge/webview2/release-notes.md]: tried a variant instead, for headings structure

File: microsoft-edge/webview2/release-notes.md (hunk 3 above)
Comment (mikehoffms) [reply microsoft-edge/webview2/release-notes.md]: pushed commit for that design

Comment (mikehoffms) [reply microsoft-edge/webview2/release-notes.md]: pushed a commit impl'g that

**Review by victorhuangwq — APPROVED** [review]
LGTM!

**Review by peiche-jessica — APPROVED** [review]

**Pull request merged**